src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -735,22 +745,22 @@ String P036_data_struct::create_display_header_text(eHeaderContent iHeaderConten
 {
   String newString, strHeader;
   const __FlashStringHelper *newString_f = F("%sysname%");
-  bool use_newString_f                   = true;
+  bool use_newString_f = true;
```

Comment:
More formatter issues, are you using VSCode with Uncrustify, as suggested & [documented](https://espeasy.readthedocs.io/en/latest/Participate/PlatformIO.html#uncrustify) in the developers guide?

Reply:
Normally, I do.
I can try the uncrustify again.

Reply:
Having differently configured formatting tools is rather annoying when reviewing the code-changes.